lib/ansible/modules/system/pamd.py

```diff
@@ -853,15 +860,9 @@ def main():
             module.fail_json(msg='Unable to create temporary \
                                     file %s' % temp_file)
 
-        module.atomic_move(temp_file.name, pamd_file)
-
-    facts = {}
-    facts['pamd'] = {'changed': changes > 0,
```

Comment:
Looks like we're no longer setting 'pamd' facts?  Were these documented?  Do we need to document removal?

Reply:
The module never documented that it was returning "facts". I removed this because the module (prior to this change) doesn't actually return things that should be considered "facts" either. It returns data about the task run, i.e. how many changes were made, etc. Those aren't facts and if you ran the module twice in the same playbook you would overwrite the facts on a host. It just doesn't make sense to store this stuff in facts vs just in return data where one would expect it.

Additionally if someone were trying to register the output of this module they would have to do some digging to figure out that instead of storing the return values as documented they would have to dip into facts to find the data.

Reply:
I'm fine w/ the change, since this was never documented, no need to mention its removal.